build-system/tasks/check-exact-versions.js

```diff
@@ -54,9 +54,7 @@ async function checkExactVersions() {
   if (success) {
     return Promise.resolve();
   } else {
-    const reason = new Error('Check failed');
-    reason.showStack = false;
-    return Promise.reject(reason);
+    return Promise.reject('Check failed');
```

Comment:
Since this is an async function, it should just be
```javascript
if (success) {
  return;
}
throw new Error('Check failed');
```

Reply:
Good catch. Simplified further to:
```js
  if (!success) {
    throw new Error('Check failed');
  }
```